extensions/ql-vscode/test/vscode-tests/cli-integration/packaging.test.ts

```diff
@@ -41,11 +41,7 @@ describe("Packaging commands", () => {
       .spyOn(helpers, "showAndLogInformationMessage")
       .mockResolvedValue(undefined);
 
-    const extension = await extensions
-      .getExtension<CodeQLExtensionInterface | Record<string, never>>(
-        "GitHub.vscode-codeql",
-      )!
-      .activate();
+    const extension = await getActivatedExtension();
     if ("cliServer" in extension) {
```

Comment:
Can we remove these checks now? It seems like the `cliServer` should always be present, otherwise `getActivatedExtension` would have thrown an error. It seems like some other tests have this same pattern as well.

Reply:
Good point! I somehow missed these checks, but they can definitely be removed. I'll also update the error message from `getActivatedExtension` to be more like this one here.